sdk/keyvault/test-resources.json

```diff
@@ -11,7 +11,7 @@
     },
     "tenantId": {
       "type": "string",
-      "defaultValue": "[subscription().tenantId]",
+      "defaultValue": "72f988bf-86f1-41af-91ab-2d7cd011db47",
```

Comment:
Why we hard code the id here?

Reply:
This was taken from JS's `test-resources.json`. I don't think there's any need to change this, so I can revert it.

Reply:
This may cause problems if we need to run the tests in another subscription.